src/lib/octokit.ts

```
@@ -26,81 +27,106 @@ const getXml = async (page: number) => {
+          )
+          break
+        }
+
+        // Add a small delay between requests to avoid rate limiting
+        if (page < MAX_PAGES) {
+          await new Promise((resolve) => setTimeout(resolve, 100))
+        }
+      } catch (pageError) {
+        console.error(`Failed to fetch page ${page}:`, pageError)
+        // Continue with next page instead of failing completely
+        continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)